### PR TITLE
Anastasia terziyan

### PR DESCRIPTION
**Pull request by AnastasiyaTerziyan**
added activeadmin, to go to the admin lokalhost: 3000/admin, added question answer model

### CODE REVIEW

File: app/models/admin_user.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class AdminUser < ActiveRecord::Base
```
Comment (ruslanshakirov) [inline app/models/admin_user.rb]: Why you create individual model for admin user?
I think it's a user role.

File: Gemfile [hunk 2]
```diff
@@ -4,6 +4,9 @@ ruby "2.3.1"
 
 gem "rails", "4.2.6"
 gem "pg"
+gem "activeadmin", github: 'activeadmin'
+gem 'inherited_resources', github: 'activeadmin/inherited_resources'
```
Comment (ruslanshakirov) [inline Gemfile]: Please use double quotes.
Why we need to use activeadmin gem?
I thought we'll create our admin panel without using other framework.

**Comment by ruslanshakirov** [comment]
Failed on bin/setup.

File: Gemfile (hunk 2 above)
Comment (AnastasiyaTerziyan) [inline Gemfile]: Impossible to install activeadmin without other frameworks

**Comment by AnastasiyaTerziyan** [comment]
I don't know why the tests are failing but i have tested manually the case which falls and it works in application

**Pull request merged**